src/Components/Common.tsx

```diff
@@ -66,9 +66,8 @@ type MessageBodyProps = {
 export const MessageBody = styled.p<MessageBodyProps>`
   display: inline-block;
   background-color: ${(props: MessageBodyProps) =>
```

Comment:
nit: I guess we don't need to declare the type here.
```suggestion
  background-color: ${(props) => {...}}
```

Reply:
Ture that! Can adress this in a new PR 🙌